language/stdlib/modules/LibraBlock.move

```diff
@@ -90,13 +86,12 @@ module LibraBlock {
         Event::emit_event<NewBlockEvent>(
             &mut block_metadata_ref.new_block_events,
             NewBlockEvent {
-                round: round,
-                proposer: proposer,
-                previous_block_votes: previous_block_votes,
+                round,
+                proposer,
+                previous_block_votes,
                 time_microseconds: timestamp,
             }
         );
-        // TODO(valerini): call regular reconfiguration here LibraSystem2::update_all_validator_info()
```

Comment:
@valerini: Just double-checking that this is an outdated comment left over from when we used to trigger reconfiguration automatically at block boundaries.

Reply:
she may be offline for a bit, if memory serves me correct, this was intended to be used for batching within a block.

Reply:
If she's not around, maybe file a GitHub issue about looking into this and then go ahead with the change. It's not like removing a comment is going to break anything....

Reply:
I remember the same as David, but I will double-check with Lera when she gets back.